desktop-app/src/components/StatsCard.jsx

```
@@ -21,18 +34,71 @@ function StatsCard({ title, value, subtitle, icon: Icon, trend, color = 'blue' }
+  const cardClasses = `bg-gradient-to-br ${colorClasses[color]} border rounded-lg p-4 transition-all duration-300 ${
+    onClick ? 'cursor-pointer hover:scale-105 hover:shadow-xl' : ''
+  } ${className}`;
+
   return (
-    <div className={`bg-gradient-to-br ${colorClasses[color]} border rounded-lg p-4`}>
+    <div 
+      className={cardClasses}
+      onClick={onClick}
+      role={onClick ? 'button' : undefined}
+      tabIndex={onClick ? 0 : undefined}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)